TestPlatform.sln

```diff
@@ -880,6 +883,18 @@ Global
 		{B6AF6BCD-64C6-4F4E-ABCA-C8AA2AA66B7B}.Release|x64.Build.0 = Release|Any CPU
 		{B6AF6BCD-64C6-4F4E-ABCA-C8AA2AA66B7B}.Release|x86.ActiveCfg = Release|Any CPU
 		{B6AF6BCD-64C6-4F4E-ABCA-C8AA2AA66B7B}.Release|x86.Build.0 = Release|Any CPU
+		{186069FE-E1E8-4DE1-BEA4-0FF1484D22D1}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
I see that we are doing the same thing for `testhost` and `testhost.x86` but I am still surprised. Why do we compile in `AnyCpu` if that's a platform specific `exe`?

Reply:
I don't know exactly. It is probably done out of habit. We ngen these when inserting into VS, but not when inserting into dotnet sdk.